appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/deployment/BeanMethodCalculatorImpl.java

```diff
@@ -92,22 +94,22 @@ public Vector getPossibleCmpCmrFields(ClassLoader cl,
             int nextModifiers = next.getModifiers();
             if( Modifier.isAbstract(nextModifiers) ) {
                 if( nextName.startsWith("get") &&
-                    nextName.length() > 3 ) {
+                        nextName.length() > 3 ) {
```

Comment:
I expect you replaced tabs with spaces, but the formatting is still weird.

Reply:
Yeah, I didn't follow up on all bad formatting and weird indenting. Maybe its better to actually run a format all on full module at a time to get those changes out of the way.